Author: Sven Joachim <svenjoac@gmx.de>
Description: Remove "rin" from screen-256color
 Some (many?) tmux users have set their TERM environment variable to
 screen-256color, and tmux before version 3.0 does not deal with "rin"
 correctly.  So drop "rin" from screen-256color, at least until tmux
 3.0 is released.
Bug-Debian: https://bugs.debian.org/933572
Last-Update: 2019-08-07

diff -wpruN '--exclude=*.orig' a~/misc/terminfo.src a/misc/terminfo.src
--- a~/misc/terminfo.src	1970-01-01 00:00:00
+++ a/misc/terminfo.src	1970-01-01 00:00:00
@@ -7346,6 +7346,7 @@ screen-16color-bce-s|GNU Screen with 16
 
 screen-256color|GNU Screen with 256 colors,
 	use=xterm+256setaf, use=screen4,
+	rin@,
 
 screen-256color-s|GNU Screen with 256 colors and status line,
 	use=xterm+256setaf, use=screen-s,
